README.rst

```diff
@@ -95,6 +95,23 @@ To turn off automatic throttling, set this to None.
 
 Check out the ``example`` directory for more information.
 
+SES Event Monitoring with Configuration Sets
+============================================
+
+You can track your SES email sending at a granular level using `SES Event
+Publishing_`.  To do this, you set up an SES Configuration Set and add event
```

Comment:
Wrapping this backtick doesn't work out well, as you can see in the diff too.

https://github.com/caltechads/django-ses/blob/2b32ccadfaf16b4ef08d099a5021eafa0f73e9c1/README.rst